frontend/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react'
+
+const useLocalStorage = (key, initialValue) => {
+   const [value, setValue] = useState(() => {
+      if (localStorage.getItem(key)) {
+         return JSON.parse(localStorage.getItem(key))
+      }
+
+      localStorage.setItem(key, JSON.stringify(initialValue))
+   })
+
+   const setStoredValue = value => {
+      setValue(value)
+
+      localStorage.setItem(key, JSON.stringify(value))
+   }
+
+   return ([value, setStoredValue])
+}
+
+export default useLocalStorage
```

Comment:
Awesome job. A different way to export this is to define it as 
`export const useLocalStorage`
The only downside is wherever you import it you have to have the same be the same since it's no longer default